tools/php-cs-fixer/composer.json

```diff
@@ -0,0 +1,5 @@
+{
```

Comment:
The entire `tools/` directory must be removed from this PR. You should store your tools outside of the project directory or use a [global gitignore file](https://docs.github.com/en/get-started/getting-started-with-git/ignoring-files#configuring-ignored-files-for-all-repositories-on-your-computer) to exclude such directories from the Git repository.